skills/lark-whiteboard/routes/mermaid.md

```
@@ -0,0 +1,27 @@
+# Mermaid 路径
+
+适用于：思维导图、时序图、类图、饼图、甘特图。
+
+## Workflow
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)